h/models/token.py

```diff
@@ -53,7 +53,6 @@ class Token(Base, mixins.Timestamps):
         sqlalchemy.Integer,
         sqlalchemy.ForeignKey("user.id", ondelete="cascade"),
         index=True,
-        nullable=True,
```

Comment:
Is non null the default?

I'm not entirely sure reading the docs. We should keep the explicit declaration IMO to avoid guessing in the future:

https://docs.sqlalchemy.org/en/20/core/metadata.html#sqlalchemy.schema.Column.params.nullable